tests/installation/addon_products_sle.pm

```diff
@@ -148,7 +148,7 @@ sub run {
     my ($self) = @_;
 
     if (get_var('SKIP_INSTALLER_SCREEN', 0)) {
-        advance_installer_window('inst-addon');
+        advance_installer_window('inst-addon') if (get_var('UPGRADE', 0) eq '0');
```

Comment:
Use check_var instead would be better. :)

Reply:
changed.